hw/ip/otp_ctrl/doc/checklist.md

```diff
@@ -10,21 +10,21 @@ stage, and updated as needed. Once done, please remove this comment before check
 This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [OTP_CTRL peripheral.]({{< relref "hw/ip/otp_ctrl/doc" >}})
 All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
 
-## Design Checklist
+<%text>## Design Checklist</%text>
```

Comment:
Do you mind also changing this from the template file?

Reply:
right, I forgot to delete that from the template. thanks.